res/css/views/elements/_ServerPicker.scss

```diff
@@ -59,7 +59,7 @@ limitations under the License.
     }
 
     .mx_ServerPicker_server {
-        color: $primary-fg-color;
+        color: #2e2f32;
```

Comment:
Instead of this explicit value, you can use `$authpage-primary-color` which always uses the light values for just this sort of problem.

Reply:
okay sure!